website/content/api-docs/secret/pki.mdx

```diff
@@ -3294,7 +3295,67 @@ $ curl \
 }
 ```
 
+### Combine CRLs From The Same Issuer
 
+This endpoint allows combining multiple different CRLs that have been signed by the
+same issuer into a single signed CRL. This is useful to generate a single authoritative
+CRL of revocations across distinct Vault clusters such as primary and performance replica
+clusters.
+
+
+| Method | Path                                   |
+|:-------|:---------------------------------------|
+| `POST` | `/pki/issuer/<issuer ref>/resign-crls` |
```

Comment:
I think we've usually used `:issuer_ref` in the past here.

Reply:
Fixed